### ISSUE TITLE
Regions have been removed from website??

### ISSUE THREAD

**Issue by Js41637** [SunDwarf/OWAPI#213]
## Issue

Blizzard seem to have removed regions from users profiles, if a user as a multi regioned account it now just links to its new id (??) but no region.

For example going to any profile with a region just redirects to you to the same page but without a region.

`https://playoverwatch.com/en-us/career/pc/us/Dad-12262`

redirects to 

`https://playoverwatch.com/en-us/career/pc/Dad-12262`

You can enter any region (or text) and it will just redirect you to their root profile.

There is no major issue here as everything still works but region options are pretty much useless as it will always return a profile no matter what region you enter making it void.

**Comment by Js41637** [comment]
For example this user has a US and an EU account and the page use to link to each other however now it only lists their two ids.

https://playoverwatch.com/en-us/career/pc/us/Sylphea-11517

![image](https://user-images.githubusercontent.com/6258213/33888454-390ac370-dfa1-11e7-8847-9c612b6a038b.png)

**Comment by SunDwarf** [comment]
> There is no major issue here as everything still works

Does it download data for all the regions, or just one?

**Comment by Js41637** [comment]
Yes your API returns data for all 3 regions except it is all the same data as any region redirects to the base page meaning they're all technically valid.

**Comment by SunDwarf** [comment]
Wait, so the data isn't region-split on the website *at all*? It's just combined?

**Comment by Js41637** [comment]
I've honestly no idea how regions work with accounts as the account I mentioned above had a `us` and `eu` region but now the page returns two separate ids as though they are separate accounts but still the same account as it links them both.

**Comment by Js41637** [comment]
So from what I can tell there is no way to tell the region of a user anymore. I assume no account (with the same id) can exist in more than 1 region so now it just links to the other accounts id omitting the region as it is irrelevant. Why they did this though I don't know.

**Comment by azah** [comment]
Dang, now I'll never know about my Korean overwatch stats.

**Comment by KingJP** [comment]
Has anyone figured out how to switch regions now?
I think Blizzard is showing the region you have played the most hours on.

**Comment by JoshCode** [comment]
The Overwatch client shows the same stats regardless of logged in region now, so my assumption is they just trashed the stats separated by region.

**Comment by relaera** [comment]
Pull request #271 has been made to address this (albeit, with a significant change to the keying of regions in the json response)
